adapters/json/json.go

```diff
@@ -0,0 +1,139 @@
+package json
+
+import (
+	"encoding/json"
+	"errors"
+	"log"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/gliderlabs/logspout/adapters/syslog"
+	"github.com/gliderlabs/logspout/router"
+)
+
+var configDefaults = map[string]string{
+	"JSON_FIELDS":          "time:uint,message,docker.hostname,docker.image",
+	"JSON_TIME":            "{{.Time.Unix}}",
+	"JSON_MESSAGE":         "{{.Data}}",
+	"JSON_SOURCE":          "{{.Source}}",
+	"JSON_DOCKER_HOSTNAME": "{{.Container.Config.Hostname}}",
+	"JSON_DOCKER_IMAGE":    "{{.Container.Config.Image}}",
+	"JSON_DOCKER_ID":       "{{.Container.ID}}",
+	"JSON_DOCKER_NAME":     "{{.ContainerName}}",
```

Comment:
This is intentional.
https://github.com/gliderlabs/logspout/blob/master/adapters/syslog/syslog.go#L135